reflex-algos/pom.xml

```diff
@@ -420,15 +420,15 @@
                                 </configuration>
                             </execution>
                             <execution>
-                                <id>package-python2.7-egg-mlcomp</id>
+                                <id>package-python2_python3-mlcomp-eggs</id>
```

Comment:
I'd rather recommend to be consistent with naming and use `py2` and `py3` to indicate python2 and python3 correspondingly

Reply:
fixed